src/config/rcxml.c

```diff
@@ -906,6 +912,19 @@ entry(xmlNode *node, char *nodename, char *content)
 		rc.window_edge_strength = atoi(content);
 	} else if (!strcasecmp(nodename, "range.snapping")) {
 		rc.snap_edge_range = atoi(content);
+	} else if (!strcasecmp(nodename, "edgeType.previewTimeout.snapping")) {
+		if (!strcasecmp(content, "interior")) {
+			edge_type = EDGE_TYPE_INTERIOR;
+		} else if (!strcasecmp(content, "exterior")) {
+			edge_type = EDGE_TYPE_EXTERIOR;
+		}
+	} else if (!strcasecmp(nodename, "previewTimeout.snapping")) {
+		if (edge_type & EDGE_TYPE_INTERIOR) {
+			rc.snap_preview_interior_timeout = atoi(content);
+		}
+		if (edge_type & EDGE_TYPE_EXTERIOR) {
+			rc.snap_preview_exterior_timeout = atoi(content);
+		}
```

Comment:
The word "timeout" here is a misnomer; I suggest the "delay" instead. Also, we'll need documentation on these user-facing options.

Reply:
Good call, another dev vs user terminology difference here.

Reply:
Done.